pfs_middleware/pfs_middleware/middleware.py

```diff
@@ -1522,6 +1522,9 @@ def get_object(self, ctx):
 
         read_plan, raw_metadata, size, mtime_ns, ino, num_writes, lease_id = \
             rpc.parse_get_object_response(object_response)
+        if req.environ.get('swift_owner') and 'get-read-plan' in req.params:
+            return swob.HTTPOk(request=req, body=json.dumps(read_plan),
+                               headers={"Content-Type": "application/json"})
```

Comment:
Maybe I should use something like `application/proxyfs-read-plan+json`? Or include a header like `X-ProxyFS-Read-Plan: True`? I kinda want _something_ to differentiate from when you're not actually looking at a bimodal account...

Reply:
Ah, yes, here you've already responded to the comment I just made :-).